packages/omg-js-util/src/transaction.js

```diff
@@ -80,21 +113,53 @@ const transaction = {
     })
   },
 
-  createTransactionBody: function (fromAddress, fromUtxos, toAddress, toAmount) {
+  /**
+  * Creates a transaction object. It will select from the given utxos to cover the amount
+  * of the transaction, sending any remainder back as change.
+  *
+  *@param {string} fromAddress the address of the sender
+  *@param {string} fromUtxos the utxos to use as transaction inputs
+  *@param {string} toAddress the address of the receiver
+  *@param {string} toAmount the amount to send
+  *@param {string} currency the currency to send
+  *@returns the transaction object
+  *@throws InvalidArgumentError if any of the args are invalid
+  *@throws Error if the given utxos cannot cover the amount
+  *
+  */
+  createTransactionBody: function (fromAddress, fromUtxos, toAddress, toAmount, currency) {
```

Comment:
I wonder how this function is going to look like when once we have UTXO management implemented

Reply:
The way I see it is that UTXO management will take the place of this function. This is really just a convenience function to use if you don't care about utxos.
But if you do select your utxos to spend via some management API, then you'll skip `createTransactionBody()` and go straight to `encode()`